.dockerignore

```diff
@@ -5,7 +5,6 @@
 
```

Comment:
(Placeholder) Address all comments on [old PR](https://github.com/edx/edx-platform/pull/24608).

Reply:
I moved the comments over here for ease of review.